src/NServiceBus.AcceptanceTests/Satellites/When_a_message_is_available.cs

```diff
@@ -56,7 +56,7 @@ protected override void Setup(FeatureConfigurationContext context)
                         {
                             var testContext = builder.Build<Context>();
                             testContext.MessageReceived = true;
-                            testContext.TransportTransactionAddedToContext = ReferenceEquals(pushContext.Context.Get<TransportTransaction>(), pushContext.TransportTransaction);
+                            testContext.TransportTransactionAddedToContext = ReferenceEquals(pushContext.Extensions.Get<TransportTransaction>(), pushContext.TransportTransaction);
```

Comment:
Yeah, that's what I meant. `PushContext` was the old name of the class, so there were variables that weren't renamed.